qiskit/circuit/classical/expr/expr.py

```diff
@@ -314,6 +314,15 @@ class Op(enum.Enum):
         container types (e.g. unsigned integers) as the left operand, and any integer type as the
         right-hand operand.  In all cases, the output bit width is the same as the input, and zeros
         fill in the "exposed" spaces.
+
+        The binary arithmetic operators :data:`ADD`, :data:`SUB:, :data:`MUL`, and :data:`DIV`
```

Comment:
```suggestion
        The binary arithmetic operators :data:`ADD`, :data:`SUB`, :data:`MUL`, and :data:`DIV`
```

Reply:
Maybe apply this in the next PR, just so I can get this one in the merge queue already?